src/test/scala/eventstore/ReadAllEventsForwardITest.scala

```diff
@@ -80,7 +83,7 @@ class ReadAllEventsForwardITest extends TestConnection {
         preparePosition = position.preparePosition - 1
       )
       readAllEventsFailed(wrongPosition, 10) must throwA[ServerErrorException]
-    }
+    }.pendingUntilFixed("It seems this behavior has changed in ES?")
```

Comment:
does it work on you local machine ?

Reply:
No, try it out. I do not see similar test in .Net client tests